objects/index.js

```diff
@@ -76,13 +76,13 @@ export const getObjectValues = (obj) => {
    should take priority; as you're copying B into A
 */
 export const joinObjects = (objA, objB) => {
-
+  return Object.assign(objA, objB);
 }
 
 /*
   I expect this function to add a isObject property onto the object passed in:
   isObject should also be set to true
 */
 export const addIsObjectProperty = (obj) => {
-
+  return obj['isObject'] = true; // I cant get this to work so I'm leaving it with what I thought should work
```

Comment:
You're on the right lines - 

You need to modify the object as you have done - but you are not returning the modified object from this function; you are returning the result from the assignment of a new property to that object

Consider the following if you will...

```js
// here we have a user object
const user = {
  name: 'Matthew',
  age: 24
};
```
This is me modifying the user object
```js
user.isObject = true;
```
The user object now has this new property associated with it
```js
console.log(user.isObject) // true
```
However, this function needs to add the property to the object and then return the new modified object